### DISCUSSION TITLE
Separate documentation website?

### DISCUSSION THREAD

**Discussion by strdr4605** [TanStack/query#327] [General] [answered]
Is there a plan in the future to move the documentation to a separate website based on https://v2.docusaurus.io/ or other tools for  OSS docs?

Also, I am working on a tool that adds links to the top of README: https://github.com/strdr4605/jump2header
so if no moving to separate docs website is planed, I can at least make a PR in future that adds links to the table of contents for easy navigation.

**Comment by tannerlinsley** [comment]
Right now, there is already a TOC in the Readme that should be sufficient. I don't have any public or immediate plans to move the docs to their own site. If I do, they would likely move to something like react-query.tannerlinsley.com, but I would need a lot of free time to do that or some significant sponsorships to justify the time it would take. 

Also for posterity, I would not use DocZ. While I think it's a pretty great solution, and I am currently trying it out for React Table, I would like to investigate a minimal documentation site built with Next.js instead.

**Comment by justincy** [comment, marked as answer]
Recently released docs site: https://react-query.tanstack.com/

**Comment by Mukhammadali** [comment]
@justincy @tannerlinsley  It would be great to see v1 on the new docs site as well. Is there any progress or plans on this?

**Reply by tannerlinsley** [reply]
No, and this likely won’t happen from myself unless it’s a community contribution.
